src/reactviews/pages/ExecutionPlan/iconMenu.tsx

```diff
@@ -0,0 +1,171 @@
+		right: 0,
+		display: "flex",
+		flexDirection: "column",
+		alignItems: "center",
+		width: "25px",
+		opacity: 1,
+		zIndex: "1",
+		position: "absolute",
+		height: "100%"
+	},
+	button: {
+		cursor: "pointer",
+		marginTop: "5px",
+		marginBottom: "5px",
+	},
+	buttonImg: {
+		display: "block"
+	},
+	seperator: {
+		width: "100%",
+		height: "2px",
+		border: "none"
+	}
+})
+
+interface IconStackProps {
+	executionPlanView: any;
+	setExecutionPlanView: any;
+	setZoomNumber: any;
+	setCustomZoomClicked: any;
+	setFindNodeClicked: any;
+	setHighlightOpsClicked:any;
+	query: any;
+}
+
+export const IconStack : React.FC<IconStackProps> = ({
+	executionPlanView,
+	setExecutionPlanView,
+	setZoomNumber,
+	setCustomZoomClicked,
+	setFindNodeClicked,
+	setHighlightOpsClicked,
+	query
+  }) => {
+	const classes = useStyles();
+	const state = useContext(ExecutionPlanContext);
+	const executionPlanState = state?.state;
+	const LocalizedConstants = executionPlanState!.localizedConstants!;
+	const [tooltipsEnabled, setTooltipsEnabled] = useState(true);
+	enum InputEnum {
+		CustomZoom,
+		FindNode,
+		HighlightOps
+	}
+
+	const handleSavePlan = async () => {
+		await state!.provider.saveExecutionPlan(executionPlanState!.sqlPlanContent!);
+	};
+
+	const handleShowXml = async () => {
+		await state!.provider.showPlanXml(executionPlanState!.sqlPlanContent!);
+	};
+
+	const handleShowQuery = async () => {
+		await state!.provider.showQuery(query);
+	};
+
+	const handleZoomIn = async () => {
+		if (executionPlanView) {
+			executionPlanView.zoomIn();
+			setExecutionPlanView(executionPlanView);
+			setZoomNumber(executionPlanView.getZoomLevel());
+		}
+	};
+
+	const handleZoomOut = async () => {
+		if (executionPlanView) {
+			executionPlanView.zoomOut();
+			setExecutionPlanView(executionPlanView);
+			setZoomNumber(executionPlanView.getZoomLevel());
+		}
+	};
+
+	const handleZoomToFit = async () => {
+		if (executionPlanView) {
+			executionPlanView.zoomToFit();
+			setExecutionPlanView(executionPlanView);
+			setZoomNumber(executionPlanView.getZoomLevel());
+		}
+	};
+
+	const handleToggleTooltips = async () => {
+		if (executionPlanView) {
+			executionPlanView.toggleTooltip();
+			setExecutionPlanView(executionPlanView);
+			setTooltipsEnabled(!tooltipsEnabled);
+		}
+	};
+
+	const setInputContainer = (inputType: InputEnum) => {
+		if (inputType == InputEnum.CustomZoom) {
+			setFindNodeClicked(false);
+			setHighlightOpsClicked(false);
+			setCustomZoomClicked(true);
+		}
+		else if (inputType == InputEnum.FindNode) {
+			setFindNodeClicked(true);
+			setHighlightOpsClicked(false);
+			setCustomZoomClicked(false);
+		}
+		else {
+			setFindNodeClicked(false);
+			setHighlightOpsClicked(true);
+			setCustomZoomClicked(false);
+		}
+	}
+
+	return (
+		<div id="iconStack" className={classes.iconStack} style={{outline: `2px solid ${utils.seperator(executionPlanState!.theme!)}`, background:`${utils.background(executionPlanState!.theme!)}`}}>

```

Comment:
@laurennat, Can we switch to toolbar from fluent ui? https://react.fluentui.dev/?path=/docs/components-toolbar--docs

This toolbar is not accessible in its current form. 

@kburtram, while it doesn't look like ads toolbar but at least it will be accessible and responsive.

Reply:
@aasimkhan30 this is a great idea, though let's do it in a subsequent PR please.

Reply:
Issue here: https://github.com/microsoft/vscode-mssql/issues/17996